src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
```

Comment:
The message 'Excavator started its work' should be more informative. It's good to include the type of machine in the message, but it should also describe the work being done in a way that's clear and specific.